georocket-server/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I try to fix the log configuration warnings before i mark the pull request as finished.